int32 id
string description
#below used only by previous order
string result 
bool success